packages/discovery-react-components/src/components/SearchFacets/SearchFacets.tsx

```diff
@@ -52,7 +52,7 @@ interface SearchFacetsProps {
   /**
    * custom handler invoked when any input element changes in the SearchFacets component
    */
-  onChange?: (e: SyntheticEvent<HTMLInputElement>) => void;
+  onChange?: (e: SyntheticEvent<HTMLInputElement> | string[]) => void;
```

Comment:
i would explain more about what `string[]` is in the javadoc here. `SyntheticEvent<HTMLInputElement>` is self-documenting. `string[]` is not